tests/booted/readonly/001-test-status.nu

```diff
@@ -5,9 +5,18 @@ tap begin "verify bootc status output formats"
 
 let st = bootc status --json | from json
 assert equal $st.apiVersion org.containers.bootc/v1
+
 let st = bootc status --json --format-version=0 | from json
 assert equal $st.apiVersion org.containers.bootc/v1
+
 let st = bootc status --format=yaml | from yaml
 assert equal $st.apiVersion org.containers.bootc/v1
 assert ($st.status.booted.image.timestamp != null)
+
+let st = bootc status --json --booted | from json
+assert equal $st.apiVersion org.containers.bootc/v1
+assert ($st.status.booted.image.timestamp != null)
+assert (($st.status | get rollback | default null) == null)
+assert (($st.status | get staged | default null) == null)
```

Comment:
This:
- ensures the booted image timestamp is present
- extracts <status>.rollback from the status, falling back to null if not found, and enforces null
- extracts <status>.staged from the status, falling back to null if not found and enforces null

Reply:
> extracts status.rollback JSON object, falling back to null if not found, and enforces null

The elements are still present just set to null, right? So I don't think we need this `| default null`.